solver/src/test/java/SqlBuilderFromSchemaTest.java

```diff
@@ -0,0 +1,234 @@
+import org.evomaster.client.java.controller.api.dto.database.schema.DbSchemaDto;
+import org.evomaster.client.java.sql.SchemaExtractor;
+import org.evomaster.client.java.sql.SqlScriptRunner;
+import org.evomaster.core.search.gene.Gene;
+import org.evomaster.core.search.gene.numeric.DoubleGene;
+import org.evomaster.core.search.gene.numeric.IntegerGene;
+import org.evomaster.core.search.gene.numeric.LongGene;
+import org.evomaster.core.sql.SqlAction;
+import org.evomaster.core.sql.SqlInsertBuilder;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class SqlBuilderFromSchemaTest {
+
+    private static Connection connection;
+
+    @BeforeAll
+    static void setup() throws Exception {
+         connection = DriverManager.getConnection("jdbc:h2:mem:constraint_test", "sa", "");
```

Comment:
are these connections ever closed? or is it possible issue of test dependencies if using same DB url in different tests? (as they would work on same DB instance)